posthog/models/activity_logging/activity_log.py

```diff
@@ -33,6 +42,12 @@ def default(self, obj):
             return obj.__dict__
         if isinstance(obj, Change):
             return obj.__dict__
+        if isinstance(obj, Merge):
+            return obj.__dict__
```

Comment:
These three `isinstance`s can be consolidated into one: `isinstance(obj, (Detail, Change, Merge))`

Reply:
Oh, that's cool!